pkgs/by-name/su/sunvox/package.nix

```diff
@@ -96,6 +105,14 @@ stdenv.mkDerivation (finalAttrs: {
     # Cleanup, make sure we didn't miss anything
     find $out/share/sunvox/sunvox -type f -name readme.txt -delete
     rmdir $out/share/sunvox/sunvox/${bindir} $out/share/sunvox/sunvox
+
+    # Resize & install icons
+    for size in 16 24 32 48 64 128 256; do
+        mkdir -p $out/share/icons/hicolor/''${size}x''${size}/apps
+        magick ${icon} -resize ''${size}x''${size} \
+        $out/share/icons/hicolor/''${size}x''${size}/apps/sunvox.png
```

Comment:
Indentation's abit wack here, but I don't know OTTOMH if we have any written-down requirements for that… I'll prolly just adjust that to my preference on the next bump or smth, no point in blocking on that.